### PR TITLE
DOCS-4461: Improvements to Shopify documentation

### PR DESCRIPTION
**Pull request by danielcivit**
This pull request includes multiple updates to the content of Shopify payment app documentation, focusing on improving clarity and updating the installation and uninstallation instructions. The most important changes include adding prerequisites, modifying installation steps, and updating the uninstallation process.

### Important Changes:

**Prerequisites and Installation:**
* Updated the prerequisites section to include country restrictions and the need for a MultiSafepay site API key.
* Modified the installation steps for better clarity and added a step to sign in to the Shopify backend.

**Uninstallation:**
* Reintroduced and updated the uninstallation steps to ensure users have the correct permissions and know how to uninstall the MultiSafepay payment app.

**Checkout Configuration:**
* Clarified the configuration settings for customer contact methods and information to ensure proper order requests.

**Miscellaneous:**
* Improved the refund process instructions for better understanding.
* Made minor adjustments to the text for better readability and consistency.
* Renaming of documents and pages declaring a stable versions, instead of beta

### CODE REVIEW

File: content/integrations/shopify-payment-apps.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-slug: 'shopify-beta'
+excerpt: "Technical manual for MultiSafepay's Shopify Payments Apps."
+slug: 'shopify-payment-apps'
 ---
 <img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Shopify.svg" width="50" align="right" style="margin: 20px; max-height: 75px"/>
 
 # Prerequisites
 
-You will need a [MultiSafepay account](/docs/getting-started-guide/).
+- [MultiSafepay account](/docs/getting-started-guide/) registered in one of the following countries: The Netherlands, Belgium, Spain, Italy.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:14]: Germany as well?

File: content/integrations/shopify-payment-apps.md [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+- A MultiSafepay [site API key](/docs/sites#site-id-api-key-and-security-code).
+- Country / currency restrictions may apply per payment app.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:16]: Note: Country ......

File: content/integrations/shopify-payment-apps.md [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
 
 # Installation
 
-To install or migrate, follow these steps:
+To install follow these steps:
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:20]: To install  Shopify Payments Apps, follow .....

File: content/integrations/shopify-payment-apps.md [hunk 4]
```diff
@@ -36,35 +39,23 @@ To install or migrate, follow these steps:
...
-5. Click on "Install" button.
+5. Click "Install".
 6. If necessary, log in to your Shopify store.
-7. On your admin page, select Install app. 
-6. Under **Settings**:
-   - **MultiSafepay API Key**: Enter your [site API key](/docs/sites#site-id-api-key-and-security-code).
-   - **Test mode**: Turn on if you are using a Test API key. Turn off for a Live API key.
-5. Click **Save**.
-6. Under **Payment configuration** click on the button **Payment configuration**
+7. On your admin page, click **Install**.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:44]: where's the admin page exactly?

File: content/integrations/shopify-payment-apps.md [hunk 5]
```diff
@@ -36,35 +39,23 @@ To install or migrate, follow these steps:
...
-   - **MultiSafepay API Key**: Enter your [site API key](/docs/sites#site-id-api-key-and-security-code).
-   - **Test mode**: Turn on if you are using a Test API key. Turn off for a Live API key.
-5. Click **Save**.
-6. Under **Payment configuration** click on the button **Payment configuration**
+7. On your admin page, click **Install**. 
+8. Under **Settings**:
+   - **MultiSafepay website API Key**: Enter your [site API key](/docs/sites#site-id-api-key-and-security-code).
+   - **MultiSafepay Environment**: Turn on if you are using a Test API key. Turn off for a Live API key.
+9. Click **Save**.
+10. Under **Payment configuration** click on the button **Payment configuration**
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:49]: configuration**,

Comment (inesmayerMSP) [reply content/integrations/shopify-payment-apps.md:49]: click the button

File: content/integrations/shopify-payment-apps.md [hunk 6]
```diff
@@ -73,25 +64,33 @@ To uninstall, follow these steps:
...
+- Customer information: We recommend setting up the shipping address phone number as required.
 
-## Reconciliation
+Note: for some payment methods, certain customer data is required, e.g. phone number is required for Amazon Pay payments.
 
-To match orders in your accounting system with your MultiSafepay account, use the MultiSafepay order ID and the Shopify payment ID.
+## Currencies
+
+Via MultiSafepay, you can only accept payments in **EUR**.
+Ensure the default currency is set to EUR under Settings > General > Store defaults > Currencies to display.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:85]: **Settings**

File: content/integrations/shopify-payment-apps.md [hunk 7]
```diff
@@ -73,25 +64,33 @@ To uninstall, follow these steps:
...
 
 ## Payment capture method
 
-Payment capture method needs to be set to **Automatically at checkout** in your Shopify settings.  This can be set up in Settings > Payments > Payment capture method.
+Payment capture method will be force to **Automatically at checkout**  when a customer submit the order request via one of these apps.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:89]: forced, submits

File: content/integrations/shopify-payment-apps.md [hunk 8]
```diff
@@ -104,11 +103,9 @@ Payment capture method needs to be set to **Automatically at checkout** in your
 1. Sign in to your Shopify backend.
 2. Go to **Orders**.
 3. Select the order you want to refund.
-4. Click on the **Refund** button.
-- Enter the refund amount.
-- Click on the **Refund** button.
-4. A refund request is sent to MultiSafepay. The refund status is updated in your Shopify backend as **pending**.
-5. The refund is processed by MultiSafepay. The refund status is updated in your Shopify backend as **refunded**.
+4. Click on the **Refund** button and enter the refund amount and confirm.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:106]: Click **Refund**, enter the amount, and confirm.

File: content/integrations/shopify-payment-apps.md [hunk 9]
```diff
@@ -119,6 +116,15 @@ Payment capture method needs to be set to **Automatically at checkout** in your
...
 </details>
 
+# Uninstallation
+
+To uninstall, follow these steps:
+
+1. Ensure that your Shopify user has <a href="https://help.shopify.com/en/manual/your-account/staff-accounts/staff-permissions/staff-permissions-descriptions#apps-and-channels-permissions" target="_blank">the permissions</a> to uninstall apps.
+2. On your admin page, go to **Settings** > **Apps and sales channels**.
+3. Select the MultiSafepay payment app you want to uninstall, and clicking on the three dots icon, select the option **Uninstall**.
+4. On the dialog window, select a reason and confirm the action by clicking in the **Uninstall** button.
```
Comment (inesmayerMSP) [inline content/integrations/shopify-payment-apps.md:126]: by clicking ** Uninstall **

File: wordlist.txt [hunk 10]
```diff
@@ -104,8 +98,10 @@ CRM
 CSRF
 CSV
 CTAP
+CTAP
```
Comment (inesmayerMSP) [inline wordlist.txt:101]: CTAP?

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

File: content/integrations/shopify-payment-apps.md (hunk 4 above)
Comment (danielcivit) [reply content/integrations/shopify-payment-apps.md:44]: The user will be redirected there, after clicking on the "install"button.

File: wordlist.txt (hunk 10 above)
Comment (danielcivit) [reply wordlist.txt:101]: It was duplicated, I just sorted the list alphabetically and added "scalability".
I just removed the duplicated one.

**Pull request merged**